zip-0225.rst

```diff
@@ -304,58 +285,58 @@ The personalization field of this hash is unmodified from ZIP 244.
-   T.5d: orchard_actions_noncompact_digest   (32-byte hash output)
-   T.5e: valueBalanceOrchard                 (64-bit signed little-endian)
+   T.4c: anchorOrchard                       (32 bytes)
+   T.4b: orchard_actions_compact_digest      (32-byte hash output)
+   T.4c: orchard_actions_memos_digest        (32-byte hash output)
+   T.4d: orchard_actions_noncompact_digest   (32-byte hash output)
+   T.4e: valueBalanceOrchard                 (64-bit signed little-endian)
 
 The personalization field of this hash is set to::
 
   "ZTxIdOrchardHash" 
 
-T.5b: orchard_actions_compact_digest
+T.4b: orchard_actions_compact_digest
 """"""""""""""""""""""""""""""""""""
 
 A BLAKE2b-256 hash of the subset of Orchard Action information intended to be included in
 an updated version of the ZIP-307 [#zip-0307]_ ``CompactBlock`` format for all Orchard
 Actions belonging to the transaction. For each Action, the following elements are included
 in the hash::
 
-   T.5b.i  : nullifier            (field encoding bytes)
-   T.5b.ii : cmx                  (field encoding bytes)
-   T.5b.iii: ephemeralKey         (field encoding bytes)
-   T.5b.iv : encCiphertext[..52]  (First 52 bytes of field encoding)
+   T.4b.i  : nullifier            (field encoding bytes)
+   T.4b.ii : cmx                  (field encoding bytes)
+   T.4b.iii: ephemeralKey         (field encoding bytes)
+   T.4b.iv : encCiphertext[..52]  (First 52 bytes of field encoding)
 
 The personalization field of this hash is set to::
 
   "ZTxIdOrcOutCHash"
 
-T.5c: orchard_actions_memos_digest
+T.4c: orchard_actions_memos_digest
 """"""""""""""""""""""""""""""""""
 
 A BLAKE2b-256 hash of the subset of Orchard shielded memo field data for all Orchard
 Actions belonging to the transaction. For each Action, the following elements are included
 in the hash::
 
-   T.5c.i: encCiphertext[52..564] (contents of the encrypted memo field)
+   T.4c.i: encCiphertext[52..564] (contents of the encrypted memo field)
 
 The personalization field of this hash is set to::
 
   "ZTxIdOrcOutMHash"
 
-T.5d: orchard_actions_noncompact_digest
+T.4d: orchard_actions_noncompact_digest
 """""""""""""""""""""""""""""""""""""""
 
 A BLAKE2b-256 hash of the remaining subset of Orchard Action information **not** intended
 for inclusion in an updated version of the the ZIP 307 [#zip-0307]_ ``CompactBlock``
 format, for all Orchard Actions belonging to the transaction. For each Action,
 the following elements are included in the hash::
 
-   T.5d.i  : cv                    (field encoding bytes)
-   T.5d.ii : rk                    (field encoding bytes)
-   T.5d.iii: encCiphertext[564..]  (post-memo suffix of field encoding)
-   T.5d.iv : outCiphertext         (field encoding bytes)
+   T.4d.i  : cv                    (field encoding bytes)
+   T.4d.ii : rk                    (field encoding bytes)
+   T.4d.iii: encCiphertext[464..]  (post-memo suffix of field encoding)
```

Comment:
```suggestion
   T.4d.iii: encCiphertext[564..]  (post-memo suffix of field encoding)
```

I think the offset may have been changed unintentionally.

Reply:
Good catch!